app/org/maproulette/framework/service/ChallengeListingService.scala

```diff
@@ -57,6 +57,13 @@ class ChallengeListingService @Inject() (repository: ChallengeListingRepository)
             List(
               // Has a task review
               BaseParameter("task_review.id", "", Operator.NULL, negate = true),
+              // Exclude unnecessary review status
+              BaseParameter(
+                "task_review.review_status",
```

Comment:
So I merged my implicit tables PR before this one. So you will probably get conflicts as this file was modified. Anyway to handle this table of column is done a little differently, and would be more like this:
```
BaseParameter(
  TaskReview.FIELD_REVIEW_STATUS,
  Task.REVIEW_STATUS_UNNECESSARY,
  negate = true,
  table = Some(TaskReview.TABLE)
)
```

Reply:
Fixed conflicts.